src/domain/Collection/BillList.js

```diff
@@ -22,6 +22,9 @@ const BillList = ({ fmk, isLoading }) => {
   const [isSave, setIsSave] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
 
+  //toggle state for autofill button
+  const [isToggled, setIsToggled] = useState(false);
```

Comment:
can you change the name to `isReset`? `isToggled` is confusing. It doesn't tell if it is 'reset' or 'autofill' until you read whole code. So it would be more clear to say `isReset` or `isAutofill`. When I see your logic, `isToggled` means `reset`.